app/src/main/java/org/mozilla/tv/firefox/channels/ChannelTile.kt

```diff
@@ -65,10 +68,40 @@ data class ChannelTile(
     }
 }
 
+sealed class ImageSetStrategy {
+    abstract operator fun invoke(imageView: ImageView)
+
+    data class ById(val id: Int) : ImageSetStrategy() {
+        override fun invoke(imageView: ImageView) {
+            imageView.setImageResource(id)
+        }
+    }
+
+    // Note that ByPath can be used with either local paths or URLs
+    data class ByPath(val path: String) : ImageSetStrategy() {
+        override fun invoke(imageView: ImageView) {
+            PicassoWrapper.client
+                // TODO find a less brittle way to retrieve this path
+                .load(path)
```

Comment:
Does loading from path not require a placeholder?

Reply:
Seems like it should, doesn't it?  But it wasn't using one before.  I'll add a TODO.